### ISSUE TITLE
avg_train_loss += 0.05 * (loss - avg_train_loss)

### ISSUE THREAD

**Issue by Beitadoge** [mkroutikov/tf-lstm-char-cnn#24]
hello~ in the "train.py" 181 line ,what is the code meaning?

**Comment by mkroutikov** [comment]
[EMA](https://en.wikipedia.org/wiki/Moving_average#Exponential_moving_average)

**Comment by Beitadoge** [comment]
@mkroutikov  Thanks O(∩_∩)O
